src/common/snippets/src/op/subgraph.cpp

```diff
@@ -36,6 +36,8 @@
 #include "snippets/lowered/pass/propagate_layout.hpp"
 #include "snippets/lowered/pass/cleanup_loop_offsets.hpp"
 #include "snippets/lowered/pass/softmax_decomposition.hpp"
+#include "snippets/lowered/pass/serialize_control_flow.hpp"
+#include "snippets/lowered/pass/serialize_data_flow.hpp"
```

Comment:
Do we need these headers here?

Reply:
To be honest, I especially did that: this is the most common file where serialization might be needed, so I decided to include serialization headers there to avoid scrolling up and write these includes each time when I want to call serialization pass during debugging. I suppose, there is no overheads for that.
If I missed something, please let me know.